package.json

```diff
@@ -78,6 +85,12 @@
           "unicorn/no-process-exit": "off"
         }
       },
+      {
+        "files": "src/**/*.test.js",
+        "rules": {
+          "camelcase": "off"
```

Comment:
I would agree the comment approach is better, however it looks like we'd be doing it a lot in tests.

I'm unsure about this in all honesty. Maybe instead of camelcase, we should disable the need of removing quotes? for instance `{'this_is_prop':'test'}`?

Reply:
Cool ... leave it as-is 👍 ... I'll just have to restrain myself from adding overrides 😈